packages/webpack-cli/src/webpack-cli.ts

```diff
@@ -294,10 +291,10 @@ class WebpackCLI implements IWebpackCLI {
     }
 
     if (needInstall) {
-      const execa = require("execa");
+      const { sync } = require("cross-spawn");
 
       try {
-        await execa(commandToBeRun, [], { stdio: "inherit", shell: true });
+        sync(packageManager, commandArguments, { stdio: "inherit" });
```

Comment:
Can't understand why this is the only spot where `sync` was not used and `shell` was set to `true`.  Could be that it was a way to make passing all the arguments in a single string work?

Reply:
Good questions, maybe it was just mistake

Reply:
Is there also a reason the shell must run async? Intuitively it does not seem the case, as there is no way the program can continue if the install does not finish.

It's true that [npm is a bash script on Unix and a cmd file on Windows](https://github.com/npm/cli/tree/latest/bin).

But later on the [comment you've linked](https://github.com/sindresorhus/execa/issues/433#issuecomment-674277826) adds 
> If you specify a shell file (as opposed to a binary), on Windows, and do not use the shell option, Execa emulates that option for you (through [the dependency node-cross-spawn](https://github.com/moxystudio/node-cross-spawn/blob/master/lib/parse.js#L36)) by basically running cmd /d /s /c /q "npm run tsc" instead of npm run tsc

I think that must be true, because f the shell option is required, the previous `execa` calls, all without `shell: true` would fail and I think someone would have reported the bug. I think the `shell: true` is really needed if you want the shell to expand or replace something in the command string.